File: 프로그래머스/2/42586. 기능개발/기능개발.py
def solution(progresses, speeds):
    answer = []
    stack = []
    for index, value in enumerate(progresses):
        if (100 - value) % speeds[index] == 0:
            stack.append((100 - value) // speeds[index])
        else:
            stack.append((100 - value) // speeds[index] + 1)
    print(stack)
    max_cnt = stack[0]
    cnt = 1
    for index, days in enumerate(stack):
        if index ==0:
            continue
        if days > max_cnt:
            max_cnt = days
            answer.append(cnt)
            cnt=1
        else:
            cnt+=1
    answer.append(cnt)
    return answer

File: 프로그래머스/2/42586. 기능개발/기능개발.js
function solution(progresses, speeds) {
    var answer = [];
    var days = [];
    for (let i=0; i<progresses.length; i++) {
        tmp = (100-progresses[i])
        if (tmp%speeds[i] !== 0)
            days.push(Math.floor(tmp/speeds[i])+1)
        else
            days.push(tmp/speeds[i])
    }
    let max_tmp = days[0]
    let cnt = 1
    for (let i=1; i<days.length; i++) {
        if (max_tmp < days[i]) {
            max_tmp = days[i]
            answer.push(cnt)
            cnt = 1
        }
        else 
            cnt = cnt+1
    }
    answer.push(cnt)
    return answer;
}